check_brevisone

```diff
@@ -130,17 +141,27 @@ def commandline(args):
                         help='The critical threshold for the minimum signal strength (in db, default -107)',
                         default=-107,
                         type=int)
-    parser.add_argument('--ssl-insecure',
-                        dest='insecure',
-                        action='store_true',
-                        default=False,
-                        help='Allow insecure SSL connections (default False)')
-    parser.add_argument('--protocol',
-                        choices=["http", "https"],
-                        default='https',
-                        help='HTTP protocol, use one of http or https (default https)')
 
-    return parser.parse_args(args)
+    parser.add_argument('--disk-warning',
+                        help='The warning threshold for the disk space (in bytes)',
+                        required=False,
+                        type=int)
+    parser.add_argument('--disk-critical',
+                        help='The critical threshold for the disk space (in bytes)',
+                        required=False,
+                        type=int)
+
+    args = parser.parse_args(args)
+
+    if args.disk_warning and not args.disk_critical:
```

Comment:
IMHO it would be better to allow only `disk_warning` or `disk_critical`. It might a be valid scenario to say "I want this to be critical every time" or "This should never be critical".
I do not see a special need to restrict someone here.

Reply:
Done